app/src/main/java/org/fossasia/pslab/experimentsetup/ExperimentErrorStrings.java

```diff
@@ -0,0 +1,27 @@
+package org.fossasia.pslab.experimentsetup;
+
+/**
+ * Created by asitava on 27/8/17.
+ */
+
+public class ExperimentErrorStrings {
+    String ERROR_MESSAGE = "Invalid Value";
```

Comment:
You can create a `static` class and define all `strings` as `static`. Then you won't need to create `ExperimentErrorStrings` object. Simply refer to `ExperimentErrorStrings.anystring`